src/test_tif_loader.cpp

```diff
@@ -68,12 +74,29 @@ class MapPublisher : public rclcpp::Node {
       }
     };
     timer_ = this->create_wall_timer(5s, timer_callback);
+    ESPG epsg;
+    Eigen::Vector3d map_origin;
+    map_->getGlobalOrigin(epsg, map_origin);
+
+    geometry_msgs::msg::TransformStamped static_transformStamped;
+    static_transformStamped.header.frame_id = map_->getCoordinateName();
+    static_transformStamped.child_frame_id = map_->getGridMap().getFrameId();
+    static_transformStamped.transform.translation.x = map_origin(0);
+    static_transformStamped.transform.translation.y = map_origin(1);
```

Comment:
```suggestion
    static_transformStamped.transform.translation.x = map_origin.x();
    static_transformStamped.transform.translation.y = map_origin.y();
```

Eigen Vector3d also supports xyz access, which lines up well with the variable it's assigned to.